.github/workflows/publish.yaml

```diff
@@ -0,0 +1,17 @@
+name: Publish JSON Schema
```

Comment:
Maybe better name is "Publish STAC schema"

Reply:
I'm not sure - what do you think about "Publish STAC JSON schema"?